blog/admin.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib import admin
+from .models import Post, User, Commentary
+from django.contrib.auth.models import Group
+
+admin.site.register(Post)
+admin.site.register(User)
+admin.site.register(Commentary)
```

Comment:
Consider using the custom admin classes (PostAdmin, UserAdmin, CommentaryAdmin) when registering models to customize the admin interface.